pkg/api/v1/types.go

```diff
@@ -2342,11 +2342,6 @@ type PodSpec struct {
 	// More info: http://releases.k8s.io/HEAD/docs/design/service_accounts.md
 	// +optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty" protobuf:"bytes,8,opt,name=serviceAccountName"`
-	// DeprecatedServiceAccount is a depreciated alias for ServiceAccountName.
-	// Deprecated: Use serviceAccountName instead.
-	// +k8s:conversion-gen=false
-	// +optional
-	DeprecatedServiceAccount string `json:"serviceAccount,omitempty" protobuf:"bytes,9,opt,name=serviceAccount"`
```

Comment:
this LGTM, do we have a convention for marking dead protobuf ids we cannot use (or are we just counting on review to prevent someone from trying to fill in a gap left by a removed field in the future)?